cmd.go

```diff
@@ -26,3 +29,13 @@ func (c CmdLinux) ListContent(path string) ([]string, error) {
 
 	return lsOutput, nil
 }
+
+func (c CmdLinux) IsDir(dir string) bool {
+	info, err := os.Stat(dir)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
When you get an error, could you return the error instead of printing it?

Reply:
Sure!

Reply:
I think that isDir should not be a blocking function, so what if we just ignore the error if it occur sometime?

Reply:
Ok, I agree